.github/actions/test_ya/action.yml

```diff
@@ -258,6 +258,29 @@ runs:
     
         .github/scripts/tests/split-junit.py -o "$JUNIT_REPORT_PARTS" "$JUNIT_REPORT_XML"
     
+    - name: Test history upload results to YDB
+      if: always()
+      continue-on-error: true
+      shell: bash
+      env:
+        PR_NUMBER: ${{ github.event.number }}
```

Comment:
env var is not used

Reply:
PR_NUMBER using here https://github.com/ydb-platform/ydb/pull/6363/files/1ca678135650e68b42388d729c84cf3e578b41e6#diff-8e41dac1844247f16cc005544180c6c5e95e58cef2c60afe65a62a2058dc85daR281